data/card_catalogue/unique-1.json

```diff
@@ -224,38 +277,50 @@
     "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "Summon a diembodied hand within 4 tiles to perform one simple action before vanishing.",
-      "flavour": "\"Magicians are a bunch of lazy bums\" - Brinley B.",
+      "rules": {
+        "active": "Evoke a disembodied hand to perform a simple action within 4 metres of you.",
+        "passive": ""
+      },
+      "flavour": "Give me five!",
       "image": "assets/core/card/illustration/Eidolimb.png"
     }
   },
   {
-    "title": "Tripwire Charm",
+    "title": "Espial Thread",
```

Comment:
- Doesn't use 'charm' since we don't know what magic is
- Doesn't use 'tripwire' since that sounds kinda clunky and too-strongly suggests a horizontal setup